fastlane/lib/fastlane/actions/download_dsyms.rb

```diff
@@ -14,10 +14,32 @@ def self.run(params)
         Spaceship::Tunes.select_team
         UI.message("Login successful")
 
+        # Get App
+        app = Spaceship::Application.find(params[:app_identifier])
+        unless app
+          UI.user_error!("Could not find app with bundle identifier '#{params[:app_identifier]}' on account #{params[:username]}")
+        end
+
+        # Process options
         version = params[:version]
         build_number = params[:build_number]
         platform = params[:platform]
+        save_directory = params[:save_directory]
+
+        # Set version if it is latest
+        if version == 'latest'
+          # Try to grab the live version first, else fallback to edit version
+          latest_version = app.live_version(platform: platform) || app.edit_version(platform: platform)
+          version = latest_version.version
+          build_number = latest_version.build_version
```

Comment:
Where is this being used besides printing it below?

Reply:
This is used to output build number, and as a check to grab the correct build when looping through them https://github.com/fastlane/fastlane/pull/10129/files/3e9a29f87fd1df907a08115b6868e97873c18c3e#diff-65dfcad8f098c3c96688110dade15ed7L42. I just hooked into the existing loop instead of short-circuiting